app/Http/Controllers/Controller.php

```diff
@@ -432,4 +439,323 @@ public function pdf()
 
         return ($stream) ? $pdf->stream() : $pdf->download('directory.pdf');
     }
+
+    public function pdfpreview()
```

Comment:
would be great to sync up on this - seems like it duplicates a lot of logic - wonder if there's a more streamlined way to achieve the same result?